Source/Core/VideoBackends/OGL/VertexManager.cpp

```diff
@@ -156,8 +158,19 @@ void VertexManager::vFlush()
   // setup the pointers
   nativeVertexFmt->SetupVertexPointers();
 
+  if (!g_Config.backend_info.bSupportsFragmentStoresAndAtomics && ::BoundingBox::active)
+  {
+    glEnable(GL_STENCIL_TEST);
```

Comment:
Which is faster, toggling this state often, or just always enable the stencil test? BoundingBox::active is just a hack, feel freel to always enable the stencil test if this is faster.
Stencil is a render-target state, which is quite expensive to validate. So I expect the state toggling to be slower for the driver and so for the CPU, but always executing the stencil test will be slower for the GPU.